app/views/vouchers/_redeem_tutor_voucher.html.erb

```diff
@@ -0,0 +1,36 @@
+<p data-cy="redeem-voucher-text">
+  <%= t('profile.become_tutor', lecture: voucher.lecture.title) %>
+
+  <% if current_user.in?(voucher.lecture.tutors) %>
+    <%= t('profile.already_tutor',
+          tutorials: tutorials_with_tutor_titles(current_user, voucher)) %>
+  <% end %>
+</p>
+
+<% if voucher.lecture.tutorials_without_tutor(current_user).any? %>
+
+  <%= render partial: "vouchers/voucher_form", locals: { voucher: voucher } %>
+
+<% elsif voucher.lecture.tutorials.any? %>
+
+    <%= t('profile.all_tutorials_taken') %>
+    <%= cancel_voucher_button %>
+
+<% else %>
+
+  <%= t('profile.no_tutorials_yet') %>
+
+  <% if current_user.in?(voucher.lecture.tutors_by_redemption) %>
+
+      <%= t('profile.already_tutor_by_redemption') %>
+      <%= cancel_voucher_button %>
+
+  <% else %>
+
+    <p class="mt-4">
+      <%= redeem_voucher_button(voucher) %>
```

Comment:
When you redeem a tutor voucher but haven't selected any tutorials, you still get the message `Your tutor status has been updated`. This might lead to confusion if users actually forgot to mark a specific tutor slot and think that everything worked fine. In my opinion, we should issue a warning instead and not allow the successfull termination of the user flow in this case.

Reply:
Should we show a warning, this should also be added to tests.

Reply:
I think being able to not select any tutorials (or talks) will be an important use case. Often, exact details for tutorials (or talks) are only known rather late, and it would be good that people can still acquire the status as tutor (or speaker), and then the teacher or an editor can assign tutorials (or talks) later when more information is known. So, in my mind, this is a feature. In a sense, the tutor status has been upgraded - from "not a tutor" to "tutor without tutorial (yet)". We might think about a more precise message to avoid confusion.

Reply:
Ah ok, yeah that makes sense. I've reworked the messages a bit in the latest commits.